sample/compose-android-app/src/main/java/com/icerockdev/MainActivity.kt

```diff
@@ -33,7 +33,7 @@ import dev.icerock.moko.permissions.PermissionsController
 import dev.icerock.moko.permissions.compose.BindEffect
 import kotlinx.coroutines.launch
 
-class MainActivity : AppCompatActivity() {
+class MainActivity : FragmentActivity() {
```

Comment:
those changes just in sample, it's not change anything in library and not fix usage of library